src/main/java/com/toamistaketracker/ToaMistake.java

```diff
@@ -5,96 +5,101 @@
+            "baba-rolling-boulder.png"),
+    BABA_FALLING_BOULDER("Ba-Ba Falling Boulder", (config) -> config.babaFallingBoulderMessage(),
+            "baba-falling-boulder.png"),
+    BABA_BANANA("Ba-Ba Banana", (config) -> config.babaBananaMessage(), "baba-banana.png"),
+    BABA_GAP("Ba-Ba Gap", (config) -> config.babaGapMessage(), "baba-gap.png"),
 
     // Wardens
-    WARDENS_P1_PYRAMID("Wardens P1 Pyramid", "I'm disco-ing!", "wardens-pyramid.png"),
-    WARDENS_P2_OBELISK("Wardens P2 Obelisk", "", "wardens-obelisk.png"),
-    WARDENS_P2_DDR("Wardens P2 DDR", "I'm dancing!", ""),
-    WARDENS_P2_WINDMILL("Wardens P2 Windmill", "I'm winded!", ""),
-    WARDENS_P2_BOMBS("Wardens P2 Bombs", "I'm getting bombed!", ""),
-    WARDENS_P2_BIND("Wardens P2 Bind", "I'm in jail!", "wardens-bind.png"),
-    WARDENS_P2_SPECIAL_PRAYER("Wardens P2 Special Prayer", "What even was that attack?", "wardens-special-prayer.png"),
-    WARDENS_P3_EARTHQUAKE("Wardens P3 Slam", "I'm tripping!", "wardens-earthquake.png"),
-    WARDENS_P3_AKKHA("Wardens P3 Akkha", "", "wardens-akkha.png"),
-    WARDENS_P3_ZEBAK("Wardens P3 Zebak", "", "wardens-zebak.png"),
-    WARDENS_P3_KEPHRI("Wardens P3 Kephri", "I'm exploding!", "wardens-kephri.png"),
-    WARDENS_P3_BABA("Wardens P3 Ba-Ba", "It's raining!", "wardens-baba.png"),
-    WARDENS_P3_LIGHTNING("Wardens P3 Lightning", "", "wardens-lightning.png"), // Too noisy
+    WARDENS_P1_PYRAMID("Wardens P1 Pyramid", (config) -> config.wardensPyramidMessage(), "wardens-pyramid.png"),
+    WARDENS_P2_OBELISK("Wardens P2 Obelisk", (config) -> "", "wardens-obelisk.png"),
+    WARDENS_P2_DDR("Wardens P2 DDR", (config) -> config.wardensDDRMessage(), ""),
+    WARDENS_P2_WINDMILL("Wardens P2 Windmill", (config) -> config.wardensWindmillMessage(), ""),
+    WARDENS_P2_BOMBS("Wardens P2 Bombs", (config) -> config.wardensBombsMessage(), ""),
+    WARDENS_P2_BIND("Wardens P2 Bind", (config) -> config.wardensBindMessage(), "wardens-bind.png"),
+    WARDENS_P2_SPECIAL_PRAYER("Wardens P2 Special Prayer", (config) -> config.wardensPrayerMessage(),
+            "wardens-special-prayer.png"),
+    WARDENS_P3_EARTHQUAKE("Wardens P3 Slam", (config) -> config.wardensSlamMessage(), "wardens-earthquake.png"),
+    WARDENS_P3_AKKHA("Wardens P3 Akkha", (config) -> "", "wardens-akkha.png"),
+    WARDENS_P3_ZEBAK("Wardens P3 Zebak", (config) -> "", "wardens-zebak.png"),
+    WARDENS_P3_KEPHRI("Wardens P3 Kephri", (config) -> config.wardensKephriMessage(), "wardens-kephri.png"),
+    WARDENS_P3_BABA("Wardens P3 Ba-Ba", (config) -> config.wardensBabaMessage(), "wardens-baba.png"),
+    WARDENS_P3_LIGHTNING("Wardens P3 Lightning", (config) -> "", "wardens-lightning.png"), // Too noisy
     ;
-
+    
     private static final Set<ToaMistake> ROOM_DEATHS = EnumSet.of(
-            DEATH_HET, DEATH_CRONDIS, DEATH_SCABARAS, DEATH_APMEKEN, DEATH_WARDENS);
-
+    DEATH_HET, DEATH_CRONDIS, DEATH_SCABARAS, DEATH_APMEKEN, DEATH_WARDENS);
```

Comment:
nit: let's keep the formatting the same. I can send you an exported formatter of what my setup is like.